sdk/containerregistry/test-resources-post.ps1

```diff
@@ -7,6 +7,13 @@ param (
 
 Import-AzContainerRegistryImage `
     -ResourceGroupName $DeploymentOutputs['CONTAINERREGISTRY_RESOURCE_GROUP'] `
-    -RegistryName $DeploymentOutputs['CONTAINERREGISTRY_USERNAME'] `
+    -RegistryName $DeploymentOutputs['CONTAINERREGISTRY_REGISTRY_NAME'] `
+    -SourceImage 'library/busybox' -SourceRegistryUri 'registry.hub.docker.com' `
+    -Mode 'Force'
+
+Import-AzContainerRegistryImage `
+    -ResourceGroupName $DeploymentOutputs['CONTAINERREGISTRY_RESOURCE_GROUP'] `
+    -RegistryName $DeploymentOutputs['CONTAINERREGISTRY_REGISTRY_NAME'] `
     -SourceImage 'library/hello-world' -SourceRegistryUri 'registry.hub.docker.com' `
-    -Mode 'Force'
\ No newline at end of file
+    -Mode 'Force' `
+    -TargetTag 'library/hello-world:test1','library/hello-world:test-delete'
```

Comment:
Where is the test1 and test-delete coming from?

Reply:
They are specified here to create the tags, these two tags are then used in the tests.